OfficialTutorials/SpaceShuttle.json

```diff
@@ -6,15 +6,17 @@
       "es": "Shuttle Approach Runway 15",
       "zh": "航天飞机返回任务-跑道15",
       "bg": "Подход за писта 15 с космическа совалка",
-      "tr": "Shuttle Approach Runway 15"
+      "tr": "Shuttle Approach Runway 15",
+      "az": "Məkkə yaxınlaşma uçuş-enmə zolağı 15"
     },
     "description": {
       "en": "Land the Space Shuttle Atlantis at Cape Canaveral by flying through the waypoints",
       "de": "Lande das Space Shuttle Atlantis in Cape Canaveral, indem du durch die Wegpunkte fliegst",
       "es": "Atteriza el Space Shuttle en Cape Canaveral volando por los waypoints",
       "zh": "飞越一系列路径点，将航天飞机降落在位于卡纳维拉尔角的肯尼迪航天中心",
       "bg": "Приземете космическата совалка \"Атлантис\" в авиобаза Кейп Канаверал, прелитайки през навигационните точки",
-      "tr": "Ara noktalardan uçarak Uzay Mekiği Atlantis'i Cape Canaveral'a indirin"
+      "tr": "Ara noktalardan uçarak Uzay Mekiği Atlantis'i Cape Canaveral'a indirin",
+      "az": "Atlantis kosmik gəmisini keçid nöqtələrindən keçərək Cape Canaveral'a endirin"
```

Comment:
This translation needs to be rewritten, apostrophes are not used to separate suffixes in Azerbaijani language